src/nodejs/lambda-handlers/submission.js

```diff
@@ -147,15 +147,31 @@ async function submitMethod(event, user) {
 }
 
 async function reviewMethod(event, user) {
-  const approvedUserPrivileges = ['ADMIN', 'REQUEST_REVIEW', 'REQUEST_REVIEW_MANAGER'];
+  const approvedUserPrivileges = ['ADMIN', 'REQUEST_REVIEW', 'REQUEST_REVIEW_MANAGER', 'CREATE_STEPREVIEW', 'REMOVE_STEPREVIEW'];
   const { id, approve } = event;
+  const userId = user.id;
   if (user.user_privileges.some((privilege) => approvedUserPrivileges.includes(privilege))) {
     const status = await db.submission.getState({ id });
     const stepType = status.step.type;
     let eventType;
+    let param;
     if (approve === 'false' || !approve) {
+      param = {
+        submission_id: id,
+        user_id: userId,
+        approve: 'rejected',
+        step_name: status.step_name
+      };
+      await db.submission.updateStatusStepReviewApproval(param);
```

Comment:
Add a break or return if this update is successful to prevent code from running unnecessarily.